### ISSUE TITLE
"error: scanner: while parsing a quoted scalar"

### ISSUE THREAD

**Issue by brandonlee503** [realm/SwiftLint#1422]
Hi everyone,

I'm having trouble setting up some basic custom rules as shown on the [lecture](https://realm.io/news/slug-jp-simard-swiftlint/). I'm running Xcode 8.3 and SwiftLint 0.18.1. The message is the title which triggers on the regexp, specifically the `\` in `"(^ *//\w+)"`

My short config is as follows:
```
disabled_rules:
  - force_cast
  - todo

custom_rules:
    comments_space: # From https://github.com/brandenr/swiftlintconfig
        name: "Space After Comment"
        regex: "(^ *//\w+)"
        message: "There should be a space after //"
        severity: error
```

Any help is greatly appreciated!

**Comment by norio-nomura** [comment]
`\` in double quote is parsed as escape sequence by YAML.
Use single quote instead of double quote:
```yaml
custom_rules:
    comments_space: # From https://github.com/brandenr/swiftlintconfig
        regex: '(^ *//\w+)'
```

You can check your configuration is valid YAML by using http://www.yamllint.com.
Syntax Highlighter in GitHub Flavored Markdown also detect invalid YAML by specifying language `yaml` to code fencing:
```yaml
custom_rules:
    comments_space: # From https://github.com/brandenr/swiftlintconfig
        regex: "(^ *//\w+)"
```

**Comment by brandonlee503** [comment]
Thank you so much!

**Comment by rresma2** [comment]
Hi guys! I was wondering if you guys can update the docs here. I was trying to implement custom rules and ran into the same issue w/ the double quotes. Using single quotes fixed the issue for me. https://github.com/realm/SwiftLint#defining-custom-rules
